SwrveSDKCommon/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.swrve.swrvesharedsdk">
-
-    <application
-        android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true">
-
-    </application>
-
+    <application android:label="@string/app_name"/>
```

Comment:
Cool! Previously we had to add separate manifest merger rule to fix this `android:allowBackup="true"`